src/rocdecode/vaapi/vaapi_videodecoder.h

```diff
@@ -97,14 +154,15 @@ class VaapiVideoDecoder {
      */
     std::unordered_map<std::string, int> gpu_uuids_to_render_nodes_map_;
```

Comment:
I see this is not static. That means each instance will create its own copy of this variable. Is this what is intended?